### ISSUE TITLE
Can BertClient.encode() also takes the tokenized sentence as input？

### ISSUE THREAD

**Issue by MrTuo** [hanxiao/bert-as-service#125]
**Prerequisites**

> Please fill in by replacing `[ ]` with `[x]`.

* [x] Are you running the latest `bert-as-service`?
* [x] Did you follow [the installation](https://github.com/hanxiao/bert-as-service#install) and [the usage](https://github.com/hanxiao/bert-as-service#usage) instructions in `README.md`?
* [x] Did you check the [FAQ list in `README.md`](https://github.com/hanxiao/bert-as-service#faq)?
* [x] Did you perform [a cursory search on existing issues](https://github.com/hanxiao/bert-as-service/issues)?

**System information**

> Some of this information can be collected via [this script](https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh).

- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): CentOS
- TensorFlow installed from (source or binary):binary
- TensorFlow version:1.12
- Python version:3.6
- `bert-as-service` version: 1.5
- GPU model and memory:P100 16G
- CPU model and memory:

---

### Description

> Please replace `YOUR_SERVER_ARGS` and `YOUR_CLIENT_ARGS` accordingly. You can also write your own description for reproducing the issue.

I'm using this command to start the server:

```bash
bert-serving-start -model_dir /tmp/english_L-12_H-768_A-12/ -num_worker=4
```

and calling the server via:
```python
bc = BertClient()
bc.encode()
```

Then this issue shows up:

...
I'm going to use BERT as word embedding (I want to concat it with my word embedding). I find that `BertClient.encode()` only takes the whole sentence as input. My tokenizer is different from yours, which cause a same sentence may have different length and I can't concat them. 
I think it is better if `BertClient.encode()`  also takes the tokenized sentence as input.

**Comment by hanxiao** [comment]
related to #124

**Comment by hanxiao** [comment]
fyi, this issue is solved and available since 1.5.2, please do 
```bash
pip install -U bert-serving-server bert-serving-client
```
Usage and more info can be found in the tutorial: https://github.com/hanxiao/bert-as-service#using-your-own-tokenizer
